java17/src/main/java/io/papermc/paperclip/Paperclip.java

```diff
@@ -1,21 +1,19 @@
 package io.papermc.paperclip;
 
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
+import java.io.*;
```

Comment:
import on demand is evil

Reply:
it's an auto imort from idea, maybe someday I'll turn it off...

Reply:
Just configure star imports to only happen when 999 imports for that path are present.